bypros/storage/sqlstore/schema/schema.sql

```diff
@@ -0,0 +1,152 @@
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+CREATE SCHEMA cothority;
+ALTER SCHEMA cothority OWNER TO bypros;
+
+SET default_tablespace = '';
+SET default_table_access_method = heap;
+
+--
+-- Migration / Versioning
+--
+
+CREATE TABLE cothority.version (
+    version_id integer
```

Comment:
no need for version_id here, right?
we can just use the database_version as primary key (there can't be 2 identical DB versions)

Reply:
ok. Should it be auto-increment, or do we let the user choose the version ?

Reply:
I'd say no auto-increment, as version management is probably best left to humans :)
the timestamp could be auto-generated though, now that I think of it :)